agent/tool-scripts/bpftrace

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# The following script runs a bpftrace program with the program supplied as argument in pbench-register-tool
+# pbench-register-tool --name=bpftrace -- --script=sample_prog.bt
+
+script_path=$(dirname $0)
+script_name=$(basename $0)
+pbench_bin="$(
+	cd ${script_path}/..
+	/bin/pwd
+)"
+
+# source the base script
+. "$pbench_bin"/base
+
+# Perftool scripts must provide the following functions
+# 1) Install the tool
+# 2) Start data collection
+# 3) Stop data collection
+# 4) post-process the data
+
+# Defaults
+tool=bpftrace
+tool_bin=/usr/bin/$tool
+group=default
+dir="/tmp"
+
+opts=$(getopt -q -o idp --longoptions "dir:,script:,group:,iteration:,start,stop,install,postprocess" -n "getopt.sh" -- "$@")
```

Comment:
s/dgi/idp/

The single character options below seem only to be `-i`, `-g`, and `-d`.